vic/drivers/classic/src/vic_classic.c

```diff
@@ -188,6 +189,18 @@ main(int   argc,
             make_in_and_outfiles(&filep, &filenames, &soil_con,
                                  &streams, dmy);
 
+            /** Reset agg_alarm for Each Stream **/
+            for (streamnum = 0;
+                 streamnum < (short int) options.Noutstreams;
```

Comment:
since `streamnum` is a `size_t` it is not clear why you are casting the right-hand-side to `(short int)`. I think you mean `(size_t)`